packages/main/src/plugin/container-registry.spec.ts

```diff
@@ -1416,7 +1425,10 @@ describe('buildImage', () => {
       containerFile: '/dir/dockerfile',
       tag: 'name',
       platform: '',
-      provider: connection,
+      provider: {
+        ...connection,
+        status: () => connection.status,
+      },
```

Comment:
IMHO the test is correct without this change

Reply:
will need another change in the BuildImageOptions like for createPod
the test should not be changed